Assets/Scripts/Scenes/InGame/Player/Item/Compass/CompassMove.cs

```diff
@@ -0,0 +1,97 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+using DG.Tweening;
+
+public class CompassMove : MonoBehaviour
+{
+    private GameObject _escapePoint;
+    private GameObject[] _Enemys;
```

Comment:
```suggestion
    private GameObject[] _enemys;
```
_escapePointが小文字始まりなので、こちらも同じようにお願いします。